Data/src/main/java/org/tribuo/data/columnar/processors/response/BinaryResponseProcessor.java

```diff
@@ -60,9 +95,66 @@ private BinaryResponseProcessor() {}
      * @param outputFactory The output factory to use.
      */
     public BinaryResponseProcessor(String fieldName, String positiveResponse, OutputFactory<T> outputFactory) {
-        this.fieldName = fieldName;
-        this.positiveResponse = positiveResponse;
+        this(Collections.singletonList(fieldName), positiveResponse, outputFactory);
+    }
+
+    /**
+     * Constructs a binary response processor which emits a positive value for a single string
+     * and a negative value for all other field values.
+     * @param fieldNames The field names to read.
+     * @param positiveResponse The positive response to look for.
+     * @param outputFactory The output factory to use.
+     */
+    public BinaryResponseProcessor(List<String> fieldNames, String positiveResponse, OutputFactory<T> outputFactory) {
+        this(fieldNames, Collections.nCopies(fieldNames.size(), positiveResponse), outputFactory);
+    }
+
+    /**
+     * Constructs a binary response processor which emits a positive value for a single string
+     * and a negative value for all other field values. the lengths of fieldNames and positiveResponses
+     * must be the same.
+     * @param fieldNames The field names to read.
+     * @param positiveResponses The positive responses to look for.
+     * @param outputFactory The output factory to use.
+     */
+    public BinaryResponseProcessor(List<String> fieldNames, List<String> positiveResponses, OutputFactory<T> outputFactory) {
+        this(fieldNames, positiveResponses, outputFactory, false);
+    }
+
+    /**
+     * Constructs a binary response processor which emits a positive value for a single string
+     * and a negative value for all other field values. the lengths of fieldNames and positiveResponses
+     * must be the same.
+     * @param fieldNames The field names to read.
+     * @param positiveResponses The positive responses to look for.
+     * @param outputFactory The output factory to use.
+     * @param displayField whether to include field names in the generated labels.
+     */
+    public BinaryResponseProcessor(List<String> fieldNames, List<String> positiveResponses, OutputFactory<T> outputFactory, boolean displayField) {
+        this(fieldNames, positiveResponses, outputFactory, "1", "0", displayField);
```

Comment:
These new constructors should probably document what they default `positiveName` and `negativeName` to, which probably implies that "1" and "0" should be public static final constants so you can easily refer to them from the Javadoc.